WebContent/WEB-INF/jsp/excelImport.jsp

```diff
@@ -277,6 +278,16 @@
 			}
 		 });
 	});
+
+    $( document ).ready(function() {
```

Comment:
Is there any scenario where it would make sense to re-enable the submit form button after it's been pressed? Would any errors or validation run once the button is hit, for example? If not, then I think this is good as-is.